.github/workflows/docs.yml

```diff
@@ -37,3 +37,14 @@ jobs:
           branch: gh-pages # The branch the action should deploy to.
           folder: docs/out # The folder the action should deploy.
           clean: true # Automatically remove deleted files from the deploy branch
+
+      - name: Algolia
```

Comment:
I think you need to call `npm run build-search` before running the Algolia step

Reply:
resolve